installer/conf/omsagent.d/squid.conf

```diff
@@ -0,0 +1,72 @@
+# Linux Squid Proxy Log Monitoring Solution for Operations Management Suite
+# Developed by Alessandro Cardoso
+# 
+# Important: This scrpt assumes that the squid.conf logformat is configured by default as per below. 
```

Comment:
That is valid data to use, but you should add unit tests that get run with Travis. See this file as an example: [in_npmd_server_plugintest.rb](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/test/code/plugins/in_npmd_server_plugintest.rb). You should make similar files called squid_log_parser_plugintest.rb and squid_stats_plugintest.rb and utilize valid example data like you gave.